.github/workflows/check-vulnerability-standalone.yml

```diff
@@ -0,0 +1,40 @@
+name: Standalone Vulnerability Scan
+
+# Make sure to pass the BRANCH name and GIT Repo name as client_payload in the dispatch event
+# Example: client_payload: { "branch": "development", "repo": "index-management" }
+# Example: client_payload: { "branch": "master", "repo": "index-management" }
+
+on:
+  repository_dispatch:
+    types: [check-vulnerability-standalone]
+
+jobs:
+  whitesource-vulnerability-scan:
+    name: WhiteSource Vulnerability Scan
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [11]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Vulnerability Scan
+        id: vulnerability_scan
+        env:
+          wss_apikey: ${{ secrets.WSS_API_KEY }}
+        run: |
+          export PATH=$JAVA_HOME:$PATH
+          cd standalone-tools ; ls -ltr 
+          gradle -v; mvn -v ; npm -v; yarn -v
+          branch=$1
+          repo=$2
```

Comment:
You can assign value like `branch=${{github.event.client_payload.branch}}` and `repo=${{github.event.client_payload.repo}` instead of $1 $2. So you dont have to hardcode it to 1st and 2nr arg

Reply:
made the change.